staging/src/k8s.io/apiextensions-apiserver/test/integration/testserver/resources.go

```
@@ -236,11 +235,11 @@ func CreateNewCustomResourceDefinition(crd *apiextensionsv1beta1.CustomResourceD
 	// before like the created RV could be too old to watch.
 	var primingErr error
 	wait.PollImmediate(500*time.Millisecond, 30*time.Second, func() (bool, error) {
-		primingErr = checkForWatchCachePrimed(crd, dynamicClientSet)
+		primingErr, retry := checkForWatchCachePrimed(crd, dynamicClientSet)
```

this shadows `primingErr` and means an actual failure gets ignored